.rhcicd/sonarqube/Dockerfile

```diff
@@ -17,43 +15,12 @@ RUN dnf --disablerepo=* -y install https://download.postgresql.org/pub/repos/yum
 RUN dnf install -y postgresql15-server
 RUN dnf install -y postgresql15-contrib
 
-
-# 1. Install the UUID package which is required for the scripts.
-# 2. Change the default "cacert" keystore's password to a custom one for more
-#    security.
-# 3. Import Red Hat IT's custom self signed certificate in the general
-#    "cacerts" file.
-# 4. Remove the certificate file.
-RUN mkdir --parents "/var/cache/yum/metadata" \
-    && microdnf install --assumeyes uuid-1.6.2-55.el9.x86_64 \
-    && microdnf clean all \
-    && readonly rh_it_root_ca_file="$(mktemp)" \
-    && readonly rh_it_root_ca_secondary_file="$(mktemp)" \
-    && curl --output "${rh_it_root_ca_file}" --insecure "${rh_it_root_ca_cert_url}" \
-    && curl --output "${rh_it_root_ca_secondary_file}" --insecure "${rh_it_root_ca_cert_secondary_url}" \
-    && keytool \
-        -cacerts \
-        -new "${cacerts_keystore_password}" \
-        -storepasswd \
-        -storepass "changeit" \
-    && keytool \
-         -alias "RH-IT-Root-CA" \
-         -cacerts \
-         -file "${rh_it_root_ca_file}" \
-         -importcert \
-         -noprompt \
-         -storepass "${cacerts_keystore_password}" \
-         -trustcacerts \
-    && keytool \
-         -alias "RH-IT-Root-CA-SECONDARY" \
-         -cacerts \
-         -file "${rh_it_root_ca_secondary_file}" \
-         -importcert \
-         -noprompt \
-         -storepass "${cacerts_keystore_password}" \
-         -trustcacerts \
-    && rm "${rh_it_root_ca_file}" \
-    && rm "${rh_it_root_ca_secondary_file}"
+# Add RedHat CAs on OS truststore (check https://certs.corp.redhat.com/ for more details)
+RUN curl --output rh_it_root_ca_file.crt --insecure "${rh_it_root_ca_cert_url}"
+RUN curl --output rh_it_root_ca_secondary_file.crt --insecure "${rh_it_root_ca_cert_secondary_url}"
+RUN cp rh*.crt /etc/pki/ca-trust/source/anchors/
+RUN update-ca-trust
+RUN rm rh*.crt
```

Comment:
I would suggest putting everything in a single `RUN` command to avoid creating new layers for each one of them.

```suggestion
# Add RedHat CAs on OS truststore (check https://certs.corp.redhat.com/ for more details)
RUN curl --output rh_it_root_ca_file.crt --insecure "${rh_it_root_ca_cert_url}" \
    && curl --output rh_it_root_ca_secondary_file.crt --insecure "${rh_it_root_ca_cert_secondary_url}" \
    && mv rh*.crt /etc/pki/ca-trust/source/anchors \
    && update-ca-trust
```